app/services/update_metadata_from_proquest_file.rb

```diff
@@ -33,7 +33,7 @@ def attributes
 
   def embargo_start_date
     if attributes[:DISS_agreement_decision_date].blank?
-      transformed_start_date
+      transformed_start_date(attributes)
```

Comment:
That change/commit was only for the sake of readability, but I'll drop it from the PR.